### PR TITLE
PCC54 hobojoe1848

### PR DESCRIPTION
**Pull request by hobojoe1848**
Difficulty level (1-10): [4]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [I had to really think about how to monitor the clipboard and copy the text from it just ONCE, ie, no immediate duplicates. It was more the thought process around it.
Got to use some while loops - for the first time in a... WHILE!]
Other feedback (what can we improve?): []

### CODE REVIEW

**Comment by hobojoe1848** [comment]
Only had the time to go super super basic on this one.
If I choose to continue it, have to throw in some sanity checks re the type of data copied. Get some unicode in there.

Maybe persistent storage... but I don't like the idea of doing that for security reasons.

File: 54/hobojoe1848/main.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+"""Keeping this simple, I needed to populate the clipboard_history list with
+a dummy entry to allow me to reference the 'last' index in the list in my print clipboard function"""
+
+clipboard_history = ["dummydata"]
+
+def print_clipboard():
+    """This function watches the clipboard and if anything NEW is added
+       it'll paste it to stdout."""
+    if pyperclip.paste() != clipboard_history[-1]:
```
Comment (pybites) [inline 54/hobojoe1848/main.py]: maybe use intermediate variable instead of calling paste 3 times?

File: 54/hobojoe1848/main.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        clipboard_history.append(pyperclip.paste())
+        print(f"""\n{pyperclip.paste()}""")
+
+if __name__ == "__main__":
+    while True:
```
Comment (pybites) [inline 54/hobojoe1848/main.py]: you can use `try...except KeyboardInterrupt` to catch a cntrl+c to exit the program ...

**Comment by pybites** [comment]
KISS, nice and simple, good work!

**Pull request merged**